helm-charts/geoserver/values.yaml

```diff
@@ -0,0 +1,18 @@
+# Default values for database-mssql.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: remiremi/geoserver
+  #'Always' because I updated the image and the new version was not pulled
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+persistence:
+  claimName: geoserver-datadir
+  accessModes:
+    - ReadWriteOnce
+  storage: 100Mi
+
+clusterUrl: "<cluster id >.kyma.shoot.live.k8s-hana.ondemand.com"
```

Comment:
I suppose you use this in api rule. API rules can be defined without cluster domain